### DISCUSSION TITLE
Zigbee2mgtt and Aqara wired wall switch

### DISCUSSION THREAD

**Discussion by Ekim167** [Koenkk/zigbee2mqtt#5323] [General]
Hello,

This is the first time I ask a question on GitHub. Do not judge strictly :)

My Home Assistant system is installed on a Raspberry Pi 4 Model B (2GB).
The Zigbee network is deployed on CC2538 and several routers (CC2531, Xiaomi ZNCZ02LM, BlitzWolf TS0121_plug) and successfully works with 20 devices (Xiaomi and TuYa).
But there are several Aqara switches (models QBKG03LM and QBKG04LM) that do not want to connect to Zigbee2MQTT (Distance to routers 3-5 meters. No messages appear in the log).
At the same time, I have 2-3 similar switches in identical conditions, which were connected the first time and work successfully (all my switches have previously worked successfully in MiHome).

I have read that changing the Zigbee channel can help.
I studied WiFi networks (my own and several surrounding neighbors - channels 1-10) and changed the Zigbee channel from 11 to 25.
After rebooting the system, I re-paired all the devices, but again the same switches do not want to be connected.

What can you advise me? Where to look for the error?

**Comment by JanHACS** [comment]
Just something to tell. Some days ago I forgot to close the joining device option on Z2M and later I saw a new device that it is not mine.  I suppose that one of my neighbours it is having the same problem than you.

**Reply by Ekim167** [reply]
:)))
I thank you for the great version!
But I don't understand what to do with this now :)

**Comment by JanHACS** [comment]
My version it is just that...a version, maybe you have others problems. And obviously I delete that device from my network, so I suppose that my neighbors could have back his device.

**Comment by drumstick77** [comment]
Hi guys,
My issue is slightly different. I've managed to paired them using zigbee2mqtt on HA. the Zigbee dongle I am using is the sonoff dongle 3.0 usb plus
So..........I’ve got some of QBKG04LM and QBKG03LM.
I had them connected to Hubitat before and all of they were showing temperature…
Now that I’ve moved them to HA, albeit no issue with pairing them, they’re NOT showing any temperature readings on the 04s

Not sure what I’m doing wrong here.
Anyone got similar issue or recommendation?

Cheers

**Reply by drumstick77** [reply]
Anyone? Please?
